### PR TITLE
Change the base image to balenalib

### PR DESCRIPTION
**Pull request by horia-delicoti**

### CODE REVIEW

File: Dockerfile.template [hunk 1]
```diff
@@ -1,11 +1,8 @@
-FROM resin/%%RESIN_MACHINE_NAME%%-debian
+FROM balenalib/%%RESIN_MACHINE_NAME%%-debian
 
 ENV INITSYSTEM on
```
Comment (imrehg) [inline Dockerfile.template]: @horia-delicoti this line does not do anything on `balenalib` base images. Likely need to remove it as well.

**Comment by imrehg** [comment]
@majorz I see that the rust test fails, due to formatting check fail. While it is not required, would that be something to fix up in some subsequent PR? or the rust tests are not in use in general yet?

File: Dockerfile.template (hunk 1 above)
Comment (horia-delicoti) [reply Dockerfile.template]: @imrehg Done.

**Comment by majorz** [comment]
@imrehg those Rust tests are some new addition to ResinCI which probably we need to remove as we already do those kinds of checks on the CircleCI front. Speaking of CircleCI it does not show on the status anymore...

**Pull request merged**